traitsui/testing/tester/ui_wrapper.py

```diff
@@ -73,6 +73,44 @@ def __init__(self, target, registries, delay=0):
         self._registries = registries
         self.delay = delay
 
+    def help(self):
+        """ Print help messages.
+        (This function is intended for interactive use.)
+        """
+        interactions = dict()
+        locations = dict()
+        for registry in self._registries[::-1]:
+            for type_ in registry.get_interactions(self.target.__class__):
+                interactions[type_] = registry.get_interaction_doc(
+                    self.target.__class__, type_)
+
+            for type_ in registry.get_locations(self.target.__class__):
+                locations[type_] = registry.get_location_doc(
+                    self.target.__class__, type_)
+
+        print("Interactions")
+        print("------------")
+        pairs = sorted(interactions.items(), key=lambda value: repr(value[0]))
```

Comment:
not sure why we are sorting by `repr(value[0])` or what `repr(value[0])` looks like.

Reply:
Because the interaction types may not support being compared.
e.g. `sorted([float, str])` will raise a TypeError.

All we care is that the help message show the objects in a deterministic order (it is easier to test). Because the help message is printing `repr(interaction_class))`, sorting them using `repr` seems reasonable, but it is really not _that_ important.